### PR TITLE
Vox take 20% more brute damage. Fragile bird bones!

### PR DESCRIPTION
**Pull request by KasparoVy**
:cl:
tweak: Vox are now 20% less resilient to brute damage due to their light, porous and flexible skeletons.
/:cl:

**As per [the Vox wiki](https://nanotrasen.se/wiki/index.php/Vox).**
"_The Vox skeletal structure is unique with its light, porous, and flexible bones_" which, although granting 'enhanced flexibility, reduced weight and increased agility (tm)', would also mean the bones are also less resilient to damage.

Based on input it was decided that a brute damage multiplier (2 lines in diff) was preferred over the per-limb bone breaking threshold definitions and per-species limb definitions (40-50 lines diff).

This has a similar effect to https://github.com/ParadiseSS13/Paradise/pull/6492 in the sense that, yes, Vox bones do indeed break easier, but now Vox die quicker too. Definitely not worst-in-class as the Greys have 25% less resilience to brute damage than Humans (5% less resilience than Vox).

Might discourage players from prolonged engagements and encourage them to flee more/fight differently, as Vox should.

### CODE REVIEW

**Comment by jughu** [comment]
Vox always are cowards thats why they have dart pistols with memechems 😊

**Comment by SamHPurp** [comment]
Obligatory:

"LORE SHOULDN'T INFLUENCE GAMEPLAY"

However I completely agree and think this is a good change.

**Comment by Necaladun** [comment]
This seems like it'll turn vox into cowards who don't ever want to get into a fair toolboxing fight with people.

I love it.

**Comment by jughu** [comment]
>throw floortile at vox
Inb4 he breaks all his bones drop on the floor shrieking

**Comment by Fruerlund** [comment]
> throw floortile at vox
Inb4 he breaks all his bones drop on the floor shrieking

Let's not be silly. This is a very nice con to all the pros and perks of playing as Vox. 👍

**Comment by KasparoVy** [comment]
It's not quite that dramatic, but you'll definitely have to be more careful.

**Comment by FlattestGuitar** [comment]
:+1:

Still, I think we should make species differentiation depend on something deeper than a bunch of numbers.

**Comment by fludd12** [comment]
I wouldn't suggest this by itself, but remind me about this when my species rework gets to the Vox.

**Comment by TravellingMerchant** [comment]
Fuck yes. This is so needed! Great work.

**Comment by lolexploit** [comment]
Remove vox eva ability, it's paying karma for a massive advantage for practically no downside since the combat is based around stuns not dps.

**Comment by Regen1** [comment]
Go for it 👍

**Comment by SheepFighter** [comment]
Is good, yaya! Voxxies is noes wantings to be fightings! Is likings this annie-tommy-call correctness!

**Comment by Lady-Luck** [comment]
>"LORE SHOULDN'T INFLUENCE GAMEPLAY"

I of course agree to an extent with this sentiment, however if gameplay does need to change to address a balance problem, the lore could certainly offer direction.

tl;dr: good change.

**Comment by tristan1333** [comment]
I'd rather bones honestly. These sort of things affect random greytiders fighting each other more than actual places where it'll matter. To enforce a playstyle (Vox are cowards, which I think is a great playstyle), I think it'd be better to go about it in a more "obvious" way than a flat damage multiplier. BUT I guess sure why not, I still don't think this will affect anyone's playstyle already. If someone's playstyle is to be affected, it's got to be a "loud" change. I'm biased for Vox and against invisible stat changes so yea.

**Comment by granodd** [comment]
A much more practical weakness than "needs to breathe N2" to offset their ability to wander into space practically naked and still be fine. Also some actual incentive to avoid physical confrontation as per their lore, too.

**Comment by KasparoVy** [comment]
@Tristan1333 I did the math and this has, on average, about as much a weakening effect to Vox bones as the other PR did. This is just that PR stepped up a notch with less code. The other PR was about a 20% nerf in bone strength.

**Comment by mrturkeytoe** [comment]
Aren't they supposed to have T H I C C  S K I I N tho? so wouldn't the other one make better sense?
